app/bin/getDescribeString.sh

```diff
@@ -5,9 +5,13 @@ if [ -z "$1" ]; then
 else
 	platform="$1";
 fi
+
+# Get the directory where the script is located
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

Comment:
It looks like you didn't finish updating the paths to be relative to the script dir